Resources/public/js/views/services/ez-contentcreateviewservice.js

```diff
@@ -56,9 +56,15 @@ YUI.add('ez-contentcreateviewservice', function (Y) {
             var content = new Y.eZ.Content(),
                 version = new Y.eZ.Version(),
                 type = this.get('contentType'),
-                defaultFields = {};
+                defaultFields = {},
+                contentTypeNames,
+                contentTypeName;
 
-            content.set('name', 'New "' + this.get('contentType').get('names')['eng-GB'] + '"');
+            contentTypeNames = this.get('contentType').get('names');
+            contentTypeName = contentTypeNames[this.get('app').get('defaultLanguageCode')]
```

Comment:
This relates somewhat to https://jira.ez.no/browse/EZP-25765 - content type names not shown when not eng-GB. May want to close/change that if we're going to fix all such cases in one go.